### PR TITLE
blm: improve information about collab summit

### PR DESCRIPTION
**Pull request by MylesBorins**
TBH I have mixed feelings here. I think that this extra copy will help
people to get involved if they want to, but I also want to make sure
it doesn't come across as supportive and not like advertising.

The intent is to resources available to folks who might want to get
involved. Multiple people who have reached out have asked for this
information... so perhaps sharing it might encourage others to
reach out as well.

### CODE REVIEW

**Comment by Trott** [comment]
Nit: Typo in commit message: `imrpove` -> `improve`

**Comment by Trott** [comment]
Nit: Also in commit message, I think `doesn't come across as supportive` should be `comes across as supportive`.

File: locale/en/index.md [hunk 1]
```diff
@@ -152,11 +152,17 @@ community. It falls on the rest of us.
...
-Node.js. Please contact us at [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org)
-or [@nodejs](https://twitter.com/nodejs) on Twitter. This effort
-will begin during or before our Collaborator Summit later this month. We will
-post more information soon.
+We have an upcoming free online [collaboration summit](https://github.com/openjs-foundation/summit#upcoming-events).
+There are a [number of different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
+to participate in across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
+We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/master/2020-06-23-Austin/orientation.md)
+to help you figure out how you might want to participate. If you have any questions about participating or
+just want to talk about getting involved please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
```
Comment (Trott) [inline locale/en/index.md]:
```suggestion
want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
```

File: locale/en/index.md [hunk 2]
```diff
@@ -152,11 +152,17 @@ community. It falls on the rest of us.
 
 ## Taking Action
 
-We will work with any Black community member who wishes to contribute to
-Node.js. Please contact us at [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org)
-or [@nodejs](https://twitter.com/nodejs) on Twitter. This effort
-will begin during or before our Collaborator Summit later this month. We will
-post more information soon.
+We have an upcoming free online [collaboration summit](https://github.com/openjs-foundation/summit#upcoming-events).
+There are a [number of different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
```
Comment (Trott) [inline locale/en/index.md]:
```suggestion
There are [many different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
```

File: locale/en/index.md [hunk 3]
```diff
@@ -152,11 +152,17 @@ community. It falls on the rest of us.
...
+to participate in across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
```
Comment (Trott) [inline locale/en/index.md]:
```suggestion
across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
```

File: locale/en/index.md [hunk 4]
```diff
@@ -152,11 +152,17 @@ community. It falls on the rest of us.
...
+We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/master/2020-06-23-Austin/orientation.md)
+to help you figure out how you might want to participate. If you have any questions about participating or
```
Comment (Trott) [inline locale/en/index.md]:
```suggestion
to help you figure out how you might want to take part. If you have any questions or
```

File: locale/en/index.md [hunk 5]
```diff
@@ -152,11 +152,17 @@ community. It falls on the rest of us.
...
+just want to talk about getting involved please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
+
+Beyond our upcoming summit we will work with any Black community member who wishes to contribute to
+Node.js core or our various Working Groups and projects.
+Please contact us at [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org)
+or [@nodejs](https://twitter.com/nodejs) on Twitter.
```
Comment (Trott) [inline locale/en/index.md]: Maybe put this paragraph first and the specifics about the summit as the second paragraph? Then we can get rid of `Beyond our upcoming summit`:

```suggestion
We will work with any Black community member who wishes to contribute to
```

**Review by Trott — APPROVED** [review]
Left a bunch of comments but looks good to me with or without my suggestions. I'm most interested in making sure we make good on the promise to update this post with information, which is what this PR accomplishes. (If we can't keep that promise, people may understandably be skeptical of the much larger promises elsewhere in this document.)

File: locale/en/index.md (hunk 5 above)
Comment (MylesBorins) [reply locale/en/index.md]: updated PTAL

**Comment by MylesBorins** [comment]
Did another pass at the copy and included a stronger commitment to work we will do moving forward

/cc @nodejs/tsc and @nodejs/community-committee PTAL

**Review by bnb — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

File: locale/en/index.md [hunk 6]
```diff
@@ -154,9 +154,20 @@ community. It falls on the rest of us.
...
+
+We have an upcoming free online [collaboration summit](https://github.com/openjs-foundation/summit#upcoming-events).
+There are [many different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
+across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
+We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/master/2020-06-23-Austin/orientation.md)
+to help you figure out how you might want to take part. If you have any questions or
+want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
+
+Moving forward we plan to offer ongoing support, mentorship, sponsorship to
+help those from the Black community to get involved in our project. We will
```
Comment (MylesBorins) [inline locale/en/index.md]:
```suggestion
help those from the Black community get involved in our project. We will
```

File: locale/en/index.md [hunk 7]
```diff
@@ -154,9 +154,20 @@ community. It falls on the rest of us.
...
+or [@nodejs](https://twitter.com/nodejs) on Twitter with any questions.
+
+We have an upcoming free online [collaboration summit](https://github.com/openjs-foundation/summit#upcoming-events).
+There are [many different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
+across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
+We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/master/2020-06-23-Austin/orientation.md)
+to help you figure out how you might want to take part. If you have any questions or
+want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
+
+Moving forward we plan to offer ongoing support, mentorship, sponsorship to
```
Comment (MylesBorins) [inline locale/en/index.md]:
```suggestion
Moving forward we plan to offer ongoing support, mentorship, and sponsorship to

```

**Pull request merged**